src/sidebars/favourites/favourite-controller.js

```diff
@@ -156,6 +158,19 @@
             });
         }
 
+        // pseudo private
+        _addError(newError) {
+            var err = this.errors, max = err.length;
+            for (var i = 0; i < max; i++) {
```

Comment:
Can this be refactored to look more like `this.errors.forEach((error) => ... )`?